botorch/optim/initializers.py

```diff
@@ -63,6 +63,73 @@
 ]
 
 
+def transform_constraints(
+    constraints: List[Tuple[Tensor, Tensor, float]], q: int, d: int
+) -> List[Tuple[Tensor, Tensor, float]]:
+    transformed = []
+    for constraint in constraints:
+        if len(constraint[0].shape) == 1:
+            transformed += transform_horizontal_constraint(constraint, d, q)
+        else:
+            transformed.append(transform_vertical_constraint(constraint, d))
+    return transformed
+
+
+def transform_horizontal_constraint(
```

Comment:
Maybe call this "intra-point" constraint instead? Or "pointwise"?

Reply:
I decided for intra-point.